Analysis/PWGDQ/include/PWGDQCore/VarManager.h

```diff
@@ -544,13 +550,30 @@ void VarManager::FillTrack(T const& track, float* values)
       }
       values[kTrackDCAxy] = track.dcaXY();
       values[kTrackDCAz] = track.dcaZ();
+
+      if (fgUsedVars[kTrackDCAsigXY])
+        values[kTrackDCAsigXY] = track.dcaXY() / std::sqrt(track.cYY());
+      if (fgUsedVars[kTrackDCAsigZ])
+        values[kTrackDCAsigZ] = track.dcaZ() / std::sqrt(track.cZZ());
+      if (fgUsedVars[kTrackDCAresXY])
+        values[kTrackDCAresXY] = std::sqrt(track.cYY());
+      if (fgUsedVars[kTrackDCAresZ])
+        values[kTrackDCAresZ] = std::sqrt(track.cZZ());
```

Comment:
The cYY() and cZZ() members are available only if the ReducedTrackBarrelCov table is available. So for this block of lines you should add an additional "if constexpr" statement to make sure the table is there, otherwise when someone will use the FillTrack function without the covariance matrix table, this will trigger a compile time error.